data/mods/Fuji_Structures/worldgen/airport/s_air_atc.json

```diff
@@ -0,0 +1,63 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "s_air_atc" ],
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        "eeeeeeeeezzzzzzzzzlllzzz",
+        "aaaaaaaaebbbbbbzzjjjjjzz",
+        "aaaaaaaaebffffbejjkkkjjz",
+        "eaaaaaaaecffgfbejkkkkkjz",
+        "eaaaaaaaebhffibejkkkkkjz",
+        "eaaaaaaaebhffibejkkkkkjz",
+        "eqqaaaaaebbbcbbejjkkkjjz",
+        "eqqaaaaaezbfffbenjjjjjzz",
+        "eqqaaaaaezbiiibenzzzzzzz",
+        "eeeaaaaaezbbbbbenzzjjjzz",
+        "zzeaaaaaeeeeeezenzjjkjjz",
+        "zzeaaaaaeeeddeeennjkkkjz",
+        "zzeaaaaaeeeddeeenzjjkjjz",
+        "zzeaaaaaeeeeeezenzzjjjzz",
+        "zzeaaaaaezzzzzzenzzzzzzz",
+        "zzeaaaaaezozppzenzzjjjzz",
+        "eeeaaaaaezozppzenzjjkjjz",
+        "aaaaaaaaezozzzzennjkkkjz",
+        "aaaaaaaaezozppzezzjjkjjz",
+        "aaaaaaaaezozppzezzzjjjzz",
+        "aaaaaaaaezozzzzzzzzzzzzz",
+        "aaaaaaaae oooooooooooooo",
+        "eeeaaaaaezzzzzzzzzzzzzzz",
+        "zzeaaaaaezzzzzzzzzzzzzzz"
+      ],
+      "terrain": {
+	    "z": [ "t_grass", "t_grass", "t_grass", "t_dirt" ],
+        "a": "t_pavement",
+        "b": "t_wall",
+        "c": "t_door_c",
+        "d": "t_radio_tower",
+        "e": "t_sidewalk",
+        "f": "t_floor",
+        "g": "t_stairs_up",
+        "h": "t_floor",
+        "i": "t_console_broken",
+        "j": "t_wall_metal",
+        "k": "t_metal_floor",
+        "l": "t_gas_pump",
+        "m": "t_water_pump",
+        "n": "t_sewage_pipe",
+        "o": "t_chainfence_v",
+        "p": "t_generator_broken",
+        "q": "t_pavement"
+      },
+      "furniture": { "h": "f_table", "q": "f_dumpster" },
+	  "gaspumps": { "l": {  } },
+      "place_loot": [
+        { "group": "office_mess", "chance": 80, "repeat": [ 3 ], "x": [ 10, 13 ], "y": [ 2, 5 ] },
+        { "group": "office_mess", "chance": 80, "repeat": [ 1 ], "x": [ 11, 13 ], "y": [ 7, 7 ] },
+        { "group": "trash", "chance": 50, "repeat": [ 5 ], "x": [ 1, 2 ], "y": [ 6, 8 ] }
+      ]
+    }
+  }
+]
```

Comment:
Missing blank line at end of this file (which is required). Same goes for most of the other files in this PR.